app/__init__.py

```diff
@@ -1,11 +1,25 @@
 from pathlib import Path
 from flask import Flask
+from flask_login import LoginManager
+
+import app.models.users as users_model
 
 
 app = Flask(__name__)
 app.config["SECRET_KEY"] = "my_precious"
 app.config["IMAGES_DIR_PATH"] = Path.cwd() / Path("app/static/images")
 
+# Install Login Manager
+login_mng = LoginManager()
+login_mng.login_view = "auth.login"
+login_mng.init_app(app)
+
+
+@login_mng.user_loader
+def load_user(user_id):
```

Comment:
I did not see where are we using this method? I saw we only used the `load_user` from the user model

Reply:
This is used at line 27 in `auth.py`. So right after all the auth checking is done and valid. This is how Flask-Login loads the authorized users.
Let me know if you have more questions on this.